### PR TITLE
update inspection: check redundant class field doc comment

### PR DESCRIPTION
**Pull request by ChrisGalliano**

### CODE REVIEW

File: src/main/kotlin/com/funivan/idea/phpClean/inspections/redundantDocCommentTag/RedundantDocCommentTagInspection.kt [hunk 1]
```diff
@@ -37,6 +39,19 @@ class RedundantDocCommentTagInspection : PhpCleanInspection() {
                 }
             }
 
+            override fun visitPhpField(field: Field) {
+                val comment = field.docComment
+                val items = mutableListOf<ParameterType>()
+                if (comment != null) {
+                    for (paramTag in comment.paramTags) {
```
Comment (funivan) [inline src/main/kotlin/com/funivan/idea/phpClean/inspections/redundantDocCommentTag/RedundantDocCommentTagInspection.kt:46]: Why you are collecting items, and then iterate over this collection. Maybe you can parameter types immediately?
```kotlin
for (paramTag in comment.paramTags) {
   checkComment(paramTag, field. declaredType)
}
```

**Review by funivan — CHANGES_REQUESTED** [review]

**Review by funivan — APPROVED** [review]
LGTM. Can I merge it?

**Comment by ChrisGalliano** [comment]
Yes. Should I add something to change notes?

**Comment by funivan** [comment]
> Yes. Should I add something to change notes?

Yes. this would be great.

**Pull request merged**